app/models/concerns/foreman_teamdynamix/host_managed_extensions.rb

```diff
@@ -0,0 +1,21 @@
+module ForemanTeamdynamix
+  module HostManagedExtensions
```

Comment:
I'm curious why you are extending HostManaged. It seems like these examples extend host instead:
https://github.com/theforeman/foreman_plugin_template/blob/master/app/models/concerns/foreman_plugin_template/host_extensions.rb

https://github.com/theforeman/foreman_cockpit/blob/master/app/models/concerns/foreman_cockpit/host_extensions.rb

Have you tried that? Any difference?

Reply:
I see now it is in fact extending Host::Managed, despite the file being called HostExtensions: https://github.com/theforeman/foreman_cockpit/blob/master/lib/foreman_cockpit/engine.rb#L38

I'm not sure why thats the convention, for them to be named differently, but lets stick to convention and name this file HostExtensions, rather than HostManagedExtensions

Reply:
updated